src/Toml/Codec/Combinator/Custom.hs

```diff
@@ -20,16 +21,21 @@ module Toml.Codec.Combinator.Custom
       -- * Validation
     , validate
     , validateIf
+
+      -- * Transform
```

Comment:
Let's rename the section to give some more meaning to it :slightly_smiling_face: 
```suggestion
      -- * With the errors reporting
```

Reply:
Sure!